### PR TITLE
added neutral embed

### PR DESCRIPTION
**Pull request by chrisj1**
add an extra embed for between 1-4 cases

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -131,15 +131,19 @@ def post_discord(
...
         "https://i.kym-cdn.com/photos/images/newsfeed/000/675/645/2c7.gif",
         "https://media.discordapp.net/attachments/783375197604413445/790625854202839100/image0.png",
+        "https://media.tenor.com/images/6603c0a47ff16ad8d3682e481e727f76/tenor.gif"
     ]
 
-    negative_thumbnails = [
+    neutral_thumbnails = [
         "https://steamcdn-a.akamaihd.net/steamcommunity/public/images/clans/5671259/7923c9b8e0a5799d4d422208b31f5ca0f4f49067.png",
         "https://static01.nyt.com/images/2020/01/28/science/28VIRUS-BATS1/28VIRUS-BATS1-videoSixteenByNineJumbo1600.jpg",
-        "https://media.tenor.com/images/6603c0a47ff16ad8d3682e481e727f76/tenor.gif",
```
Comment (elihschiff) [inline main.py:139]: why did you remove this image?

**Review by elihschiff — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (elihschiff) [reply main.py:139]: oh wait you added it above, nevermind

Comment (chrisj1) [reply main.py:139]: smh ELi

**Comment by johnnyapol** [comment]
thanks for the PR!
